util/testutil/integration/sandbox.go

```diff
@@ -322,7 +337,15 @@ func CheckFeatureCompat(t *testing.T, sb Sandbox, reason ...string) {
 	var ereasons []string
 	for _, r := range reason {
 		if _, ok := features[r]; ok {
-			if b.isUnsupportedFeature(r) {
+			envFeatureName := "BUILDKIT_TEST_FEATURE_" + strings.ToUpper(r)
```

Comment:
Alternative could be a single env-var, but a comma-separated list of features 🤔  `BUILDKIT_TEST_FEATURE=a,b,c,d` and/or `BUILDKIT_TEST_DISABLE_FEATURE` 🤔